app/javascript/app/components/no-content/no-content-component.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import styles from './no-content-styles.scss';
+
+const NoContent = ({ className, message }) =>
+  (<div className={cx(styles.noContent, className)}>
+    {message}
```

Comment:
I'm missing this https://www.grida.no/assets/monete-2e1c5ee7fb9ae7ca5cadde316a9ad8412b61ee97257ba6f419cd643f1f2f4493.png here 🙈